src/ethereum/index.js

```diff
@@ -69,6 +69,7 @@ export function loadContracts() {
         'Update',
         'Transfer',
         'UpdateOperator',
+        'UpdateManager',
         'Approval',
         'ApprovalForAll'
       ],
```

Comment:
I know it is not part of this PR, but `Transfer` now has 3 types with that version of dcl-eth.
Is it ok for you to update it here?